### PR TITLE
[RA1 Ch01] Acronyms added

### PR DESCRIPTION
**Pull request by karinesevilla**
For Chapter 8 Conformance

### CODE REVIEW

File: doc/ref_arch/openstack/chapters/chapter01.rst [hunk 1]
```diff
@@ -163,12 +163,20 @@ Abbreviations
      - Definition
    * - API
      - Application Programming Interface
+   * - BGP VPN
+     - Border gateway Protocol Virtual Private network
+   * - CI/CD
+     - Continuous Integration/ Continuous Deployment
```
Comment (collivier) [inline doc/ref_arch/openstack/chapters/chapter01.rst:169]: blank char?

Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter01.rst]: Okay, removed

**Review by pgoyal01 — APPROVED** [review]

**Review by collivier — APPROVED** [review]

**Comment by CsatariGergely** [comment]
@collivier @karinesevilla  This pr had only two approvals. Why was this already merged?

**Comment by karinesevilla** [comment]
@CsatariGergely This PR was considered as minor, as it is just the addition of missing acronyms.

**Comment by collivier** [comment]
@CsatariGergely you should check instead the direct pushes to master without PR. It obscures the process review (I could  be ok for a self "nit" PR merge) but it forbids the job verification and introduces new bugs to all substreams at the end.

**Comment by CsatariGergely** [comment]
@collivier @karinesevilla I understand that you are in a rush and I think it is important to finish the GSMA work in time. Also I think it is important to follow the processes we agreed on. (Some more philosophy on this [here](https://github.com/cntt-n/CNTT/pull/3154#issuecomment-1242044952))
I totally agree that we should not push anything to master without a pr. @wmk-admin please follow the pr procedure defined [here](https://github.com/cntt-n/CNTT/blob/master/CONTRIBUTING.rst).

**Comment by collivier** [comment]
@CsatariGergely  I would have expected more contributions than policies. I don't see any change proposal from all your comments counting reviewer. **TSC Co Chair activities is about the meeting itself noting more** -
Even if I generalize to the TSC activities (what's false here), TSC should help the project to be successful not the opposite.

Comparing direct commit and PR with less than 3 reviewer is technically speaking fully false.

@karinesevilla Would you mind adding a slot for my concerns about the current tracking to the next RA1 meeting?
We can hardly be successful if we do reach unreachable criteria.

**Pull request merged**